src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -174,6 +174,7 @@ Sentry.init({
     integrations: [
         new ApmIntegrations.Tracing(),
     ],
+    tracesSampleRate: 1, // To enable sending traces
```

Comment:
```suggestion
    tracesSampleRate: 0.1,
```

I suggest:

- Using floats to help emphasize `tracesSampleRate` is a (rational) number in the [0.0, 1.0] range.
- Never set it to `1.0`, as it is often a bad idea for production traffic. Can be useful in development for debugging.